primeqa/pipelines/extractive_mrc_pipeline.py

```diff
@@ -0,0 +1,99 @@
+from transformers import AutoConfig, AutoTokenizer
+from primeqa.mrc.models.heads.extractive import EXTRACTIVE_HEAD
+from primeqa.mrc.models.task_model import ModelForDownstreamTasks
+from primeqa.mrc.processors.preprocessors.tydiqa import TyDiQAPreprocessor
+from primeqa.mrc.processors.preprocessors.base import BasePreProcessor
+
+from transformers import DataCollatorWithPadding
+from primeqa.mrc.processors.postprocessors.extractive import ExtractivePostProcessor
+from primeqa.mrc.processors.postprocessors.scorers import SupportedSpanScorers
+from transformers import TrainingArguments
+
+
+from primeqa.mrc.trainers.mrc import MRCTrainer
+from datasets import Dataset
+import json
+
+class MRCPipeline():
+    
+    def __init__(self,model_for_qa):
+        task_heads = EXTRACTIVE_HEAD
```

Comment:
task_heads should be obtained from input arguments. 'EXTRACTIVE_HEAD' can be used as the default value. Now there are two task_heads ready for use.